simple_diarizer/diarizer.py

```diff
@@ -229,10 +235,10 @@ def diarize(
                             'cluster_labels': cluster_labels (list): cluster label for each embed in embeds
                             }
 
-        Uses AHC/SC to cluster
+        Uses AHC/SC/NME-SC to cluster
         """
         recname = os.path.splitext(os.path.basename(wav_file))[0]
-
+        outfile=recname+'.rttm'
```

Comment:
No!
You are overriding the function input.
You should define this something else (probably in your main.py)

Also remove the change around line 272/279 (`if outfile: ...`)

Reply:
removed